docs/source/guides/saving_and_loading_dataframes.ipynb

```diff
@@ -0,0 +1,275 @@
+    "\n",
+    "- the version of the schema at the time of saving the DataFrame\n",
+    "- the DataFrame name specified by `DataFrame.ww.name`\n",
+    "- the column names for the index and time index\n",
+    "- the column typing information, which contains the logical types with their parameters and semantic tags for each column\n",
+    "- the loading information required for the DataFrame type and file format\n",
+    "- the table metadata provided by `DataFrame.ww.metadata` (must be JSON serializable)\n",
+    "\n",
+    "```text\n",
+    "{\n",
+    "    \"schema_version\": \"10.0.2\",\n",
+    "    \"name\": \"demo_retail_data\",\n",
+    "    \"index\": \"order_product_id\",\n",
+    "    \"time_index\": \"order_date\",\n",
+    "    \"column_typing_info\": [...],\n",
+    "    \"loading_info\": {\n",
+    "        \"table_type\": \"pandas\",\n",
+    "        \"location\": \"data/demo_retail_data.csv\",\n",
+    "        \"type\": \"csv\",\n",
+    "        \"params\": {\n",
+    "            \"compression\": null,\n",
+    "            \"sep\": \",\",\n",
+    "            \"encoding\": \"utf-8\",\n",
+    "            \"engine\": \"python\",\n",
+    "            \"index\": false\n",
+    "        }\n",
+    "    },\n",
+    "    \"table_metadata\": {}\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "## Loading a Woodwork DataFrame\n",
+    "\n",
+    "After saving a Woodwork DataFrame, you can load the DataFrame and typing information by using `woodwork.deserialize.read_woodwork_table`. This function will use the stored typing information in the specified directory to recreate the Woodwork DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "oriental-baking",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from woodwork.deserialize import read_woodwork_table\n",
+    "df = read_woodwork_table('retail')\n",
+    "df.ww.schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dependent-liberty",
+   "metadata": {},
+   "source": [
+    "### Loading the DataFrame and typing information separately\n",
+    "\n",
+    "You can also load the Woodwork DataFrame and typing information separately by using `woodwork.read_file`. This function uses the `content_type` parameter to determine the file format. If `content_type` is not specified, the function will try to infer the file format from the file extension. The typing information such as the index, time index, logical types, and semantics tags are optional parameters. This approach is helpful if you want to save and load the typing information outside the specified directory or read a data file directly into a Woodwork DataFrame. To illustrate, I will write a pandas DataFrame to a single file in different formats, then load it into a Woodwork DataFrame using this typing information."
```

Comment:
Can we interweave the example with this paragraph?  We can demonstrate the basics of how it works before going over all the specifics

Reply:
yes, updated